Community/gsoc-alumn.md

```diff
@@ -0,0 +1,108 @@
+# CHAOSS and Google Summer of Code Alumni
+
+## CHAOSS GSoC students in 2019
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Parth Sharma](#parth-sharma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
+|[Bingwen Ma](#bingwen-ma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
+|[Aniruddha Karajgi](#aniruddha-karajgi)|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Nishchith K Shetty](#nishchith-k-shetty)|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+
+## CHAOSS GSoC students in 2018
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Keanu Nichols](#keanu-nichols)|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
+
+
+
+## Parth Sharma
+
+### Project Title 
+Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur
+### Project Description
```

Comment:
Please be mindful about spacing around headings.

In our CHAOSS community handbook, we have the rule to have two empty lines before and one after each heading.

```suggestion


### Project Description

```

Reply:
reference: https://github.com/chaoss/governance/blob/master/community-handbook/handbook-usage.md